### PR TITLE
Updated teams layout to include members & removed members page

### PR DESCRIPTION
**Pull request by jenetlan**

### CODE REVIEW

File: ServerCore/Pages/Puzzles/SubmitFeedback.cshtml.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 namespace ServerCore.Pages.Puzzles
 {
-//    [Authorize(Policy = "PlayerIsOnTeam, PlayerCanSeePuzzle")] // TODO: These auth checks not working currently.
+    [Authorize(Policy = "PlayerIsOnTeam, PlayerCanSeePuzzle")] // TODO: These auth checks not working currently.
```
Comment (lambertwang) [inline ServerCore/Pages/Puzzles/SubmitFeedback.cshtml.cs]: Are they still not working correctly?

File: ServerCore/Helpers/TeamHelper.cs [hunk 2]
```diff
@@ -39,5 +55,54 @@ public static async Task DeleteTeamAsync(PuzzleServerContext context, Team team)
 
             await context.SaveChangesAsync();
         }
+
+        public static async Task<Tuple<bool, string>> AddMemberAsync(PuzzleServerContext context, Event Event, EventRole EventRole, int teamId, int userId, int applicationId)
```
Comment (lambertwang) [inline ServerCore/Helpers/TeamHelper.cs]: Could you add a function comment block describing the parameters and return values?

File: ServerCore/Pages/Teams/Details.cshtml.cs [hunk 3]
```diff
@@ -36,15 +43,93 @@ public async Task<IActionResult> OnGetAsync(int teamId = -1)
 
             if (Team == null)
             {
-                if (EventRole != ModelBases.EventRole.play)
+                return NotFound("No team found with id '" + teamId + "'.");
+            }
+
+            // Existing team members
+
```
Comment (lambertwang) [inline ServerCore/Pages/Teams/Details.cshtml.cs]: Extra blank line

File: ServerCore/Pages/Teams/Details.cshtml [hunk 4]
```diff
@@ -4,58 +4,217 @@
...
+                            </td>
+                            <td>
+                                @Html.DisplayFor(modelItem => item.Member.Email)
+                            </td>
+                            <td>
+                                @if (item.Member.EmployeeAlias == null)
+                                {
+                                    <p>N/A</p>
+                                }
+                                @Html.DisplayFor(modelItem => item.Member.EmployeeAlias)
```
Comment (lambertwang) [inline ServerCore/Pages/Teams/Details.cshtml]: Else?

File: ServerCore/Pages/Teams/Details.cshtml [hunk 5]
```diff
@@ -4,58 +4,217 @@
...
+                            </td>
+                            @if (canAdd)
```
Comment (lambertwang) [inline ServerCore/Pages/Teams/Details.cshtml]: The ability to remove members should be tracked by a different variable.

File: ServerCore/Pages/Teams/Details.cshtml.cs [hunk 6]
```diff
@@ -36,15 +43,93 @@ public async Task<IActionResult> OnGetAsync(int teamId = -1)
...
+            StringBuilder emailList = new StringBuilder("");
+            foreach (TeamMembers Member in Members)
+            {
+                emailList.Append(Member.Member.Email + "; ");
+            }
+            Emails = emailList.ToString();
+
+            // Team applicants
+
```
Comment (lambertwang) [inline ServerCore/Pages/Teams/Details.cshtml.cs]: Extra blank line

File: ServerCore/Helpers/TeamHelper.cs (hunk 2 above)
Comment (lambertwang) [inline ServerCore/Helpers/TeamHelper.cs]: We should remove all pending applications when the last member of a team is added and the team is full.

File: ServerCore/Pages/Puzzles/SubmitFeedback.cshtml.cs (hunk 1 above)
Comment (jenetlan) [reply ServerCore/Pages/Puzzles/SubmitFeedback.cshtml.cs]: Re-commented the line - thanks :)

File: ServerCore/Helpers/TeamHelper.cs (hunk 2 above)
Comment (lambertwang) [reply ServerCore/Helpers/TeamHelper.cs]: #261

Comment (jenetlan) [reply ServerCore/Helpers/TeamHelper.cs]: Added :)

File: ServerCore/Pages/Teams/Details.cshtml.cs (hunk 3 above)
Comment (jenetlan) [reply ServerCore/Pages/Teams/Details.cshtml.cs]: Removed

File: ServerCore/Pages/Teams/Details.cshtml (hunk 4 above)
Comment (jenetlan) [reply ServerCore/Pages/Teams/Details.cshtml]: Fixed - thanks for catching :)

File: ServerCore/Pages/Teams/Details.cshtml (hunk 5 above)
Comment (jenetlan) [reply ServerCore/Pages/Teams/Details.cshtml]: Done - changed to check if admin or event member changes were active

File: ServerCore/Pages/Teams/Details.cshtml.cs (hunk 6 above)
Comment (jenetlan) [reply ServerCore/Pages/Teams/Details.cshtml.cs]: Removed

**Pull request merged**